packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -29,6 +29,13 @@ export interface QueueProcessingServiceBaseProps extends BaseProps {
    */
   readonly enableLogging?: boolean;
 
+  /**
+   * The environment variables to pass to the container.
+   *
```

Comment:
Is this really the default or will it always be passed?

Reply:
This part is not mine, it seems that it's always passed, I can update the doc.

Reply:
Thanks, that would be nice